script/setup-ci

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-
-set -eox pipefail
-
-bin/rails assets:precompile
-
-# download the codeclimate test reporter
-curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-chmod +x ./cc-test-reporter
-
-./cc-test-reporter before-build
-
-# Download the ChromeDriver matching the Chrome version installed by heroku-buildpack-google-chrome.
-RAILS_ENV=test bundle exec rake webdrivers:chromedriver:update WD_CHROME_PATH=$GOOGLE_CHROME_SHIM
```

Comment:
Maybe we start getting mismatched versions because this has not been duplicated in the Github CI?